### ISSUE TITLE
quadpy

### ISSUE THREAD

**Issue by nschloe** [inducer/modepy#5]
Perhaps, the quadrature code can be outsourced to quadpy. https://github.com/nschloe/quadpy

**Comment by inducer** [comment]
TIL about quadpy. That looks great! Thanks for pointing it out. I would adopt it in a heartbeat, but my whole stack is MIT, and purposefully so.

**Comment by inducer** [comment]
Do you think a license change for quadpy is in the cards?

**Comment by nschloe** [comment]
I'll think about it. Let's close this issue for now.

**Comment by inducer** [comment]
Thanks for considering it.

I don't know if it's helpful, but I'll outline my thought process for going MIT-only, which I arrived at after discussing with many folks, including @knepley (who co-maintains PETSc). Ultimately, numerics can hope to have a twofold impact: in science and in industry.

For science, the license discussion is mostly moot: There is no lever in the GPL to force folks who don't publish their science code to do so, as published papers aren't "products" covered by the GPL.

For industry, the GPL, from the point of view of corporate lawyers, poses an untenable risk. Suppose we acknowledge that intellectual property is a thing. Then any use of GPL software (and subsequent, even if unintentional) release of a product involving both valuable IP and GPL software risks instant devaluation of said IP. Understandably, industry will steer clear.

Are you okay with zero direct industry impact? I'm not.

In addition, it seems to me that (at least some) human beings tend to do better with social norms than coercion. Generally, I've observed much more outside contribution to my liberally-licensed code.
